client/components/drop-zone/index.jsx

```diff
@@ -251,7 +253,8 @@ export const DropZone = React.createClass( {
 			'is-active': this.state.isDraggingOverDocument || this.state.isDraggingOverElement,
 			'is-dragging-over-document': this.state.isDraggingOverDocument,
 			'is-dragging-over-element': this.state.isDraggingOverElement,
-			'is-full-screen': this.props.fullScreen
+			'is-full-screen': this.props.fullScreen,
+			[ this.props.customClass ]: true,
```

Comment:
Might as well place this as a direct argument of `classNames`, right after `'drop-zone'`